sanic/config.py

```diff
@@ -1,10 +1,75 @@
+from sanic.defaultFilter import DefaultFilter
 import os
-
 import types
 
 SANIC_PREFIX = 'SANIC_'
 
 
+DEFAULT_LOG_CONF = {
```

Comment:
Are we going to make sanic a Django-like project? If that's true, I will follow this naming.